bin/apps.bash

```diff
@@ -76,32 +74,30 @@ apps_sc() {
     # at least Helmfile does "just in time" template validation. Not as nice,
     # but feels good enough until we figure out something smarter.
     #
-    #[ "$1" != "--skip-template-validate" ] && template_validate_sc
```

Comment:
Keep this comment, it was commented out because we currently can't use it but not necessarily something we want to have disabled forever.